app/scripts/controllers/transactions/index.js

```diff
@@ -1361,10 +1205,8 @@ export default class TransactionController extends EventEmitter {
    *
    * @param {number} txId - the tx's Id
    * @param {string} actionId - actionId passed from UI
-   * @param opts - options object
-   * @param opts.hasApprovalRequest - whether the transaction has an approval request
    */
-  async approveTransaction(txId, actionId, { hasApprovalRequest = true } = {}) {
+  async approveTransaction(txId, actionId) {
```

Comment:
This is a weird one but... now that this is not called externally via `TransactionController` instance, but only via `PendingTransactionTracker` via passing the method in the constructor... should it be private?

Reply:
I didn't notice `PendingTransactionTracker` was constructed inside this controller, so changed to private to encapsulate everything.